src/host/enclave.h

```diff
@@ -56,6 +70,8 @@ namespace host
       }
       else
       {
+        oe_log_set_callback(nullptr, nop_oe_logger);
```

Comment:
Can we keep oe logging when VERBOSE_LOGGING is on?

Reply:
What's the resolution here?

Reply:
I missed this - will disable this nop logger when `VERBOSE_LOGGING` is on so we restore the old behaviour. I believe that's still a single line though, unless apps can turn on more OE logging themselves?